src/test/java/net/masterthought/cucumber/ReportParserTest.java

```diff
@@ -88,4 +90,145 @@ public void parseJsonResults_OnNoExistingFile_ThrowsException() {
 
         reportParser.parseJsonFiles(jsonReports);
     }
+
+    @Test
+    public void parsePropertyFiles_EmptyProperties() {
+
+        // given
+        initWithProperties(EMPTY_PROPERTIES);
+        ReportParser reportParser = new ReportParser(configuration);
+
+        // when
+        reportParser.parsePropertiesFiles(propertyReports);
+
+        // then
+        assertThat(configuration.getClassifications()).isEmpty();
+    }
+
+    @Test
+    public void parsePropertyFiles_Populate_One_File() {
+
+        // given
+        initWithProperties(SAMPLE_ONE_PROPERTIES);
+        ReportParser reportParser = new ReportParser(configuration);
+
+        // when
+        reportParser.parsePropertiesFiles(propertyReports);
+
+        // then
+        assertThat(configuration.getClassifications()).hasSize(5);
+    }
+
+    @Test
+    public void parsePropertyFiles_Populate_Two_Files() {
+
+        // given
+        initWithProperties(SAMPLE_ONE_PROPERTIES, SAMPLE_TWO_PROPERTIES);
+        ReportParser reportParser = new ReportParser(configuration);
+
+        // when
+        reportParser.parsePropertiesFiles(propertyReports);
+
+        // then
+        assertThat(configuration.getClassifications()).hasSize(8);
+    }
+
+    @Test
+    public void parsePropertyFiles_Populate_Check_Contents_Integrity() {
+
+        // given
+        initWithProperties(SAMPLE_TWO_PROPERTIES);
+        ReportParser reportParser = new ReportParser(configuration);
+
+        // when
+        reportParser.parsePropertiesFiles(propertyReports);
+
+        List<Map.Entry<String, String>> classifications = configuration.getClassifications();
+
+        // then
+        assertThat(classifications).hasSize(3);
+        assertThat(classifications).containsExactly(entry("NodeJsVersion","8.5.0"),entry("Proxy", "http=//172.22.240.68:18717"),entry("NpmVersion", "5.3.0"));
+    }
+
+    @Test
+    public void parsePropertyFiles_Populate_Check_Contents_Order() {
+
+        // given
+        initWithProperties(SAMPLE_TWO_PROPERTIES);
+        ReportParser reportParser = new ReportParser(configuration);
+
+        // when
+        reportParser.parsePropertiesFiles(propertyReports);
+
+        List<Map.Entry<String, String>> classifications = configuration.getClassifications();
+
+        // then
+        assertThat(classifications).hasSize(3);
+        assertThat(classifications.get(0).getKey()).isEqualTo("NodeJsVersion");
```

Comment:
can you use `containsExactly` like above?

Reply:
Yes agreed...
there is two tests...
```
parsePropertyFiles_Populate_Check_Contents_Integrity
parsePropertyFiles_Populate_Check_Contents_Order
```
both use the same test data. ContainsExactly checks content integrity and order so I'm going to delete one of these methods & rename kept one to
```
parsePropertyFiles_Populate_Check_Content_Integrity_And_Order
```
agreed?

Reply:
ok